packages/modular-scripts/src/__tests__/app.test.ts

```diff
@@ -143,14 +143,16 @@ describe('When working with a nested app', () => {
          ├─ css
          │  ├─ main.1a7488ce.css #x701i6
          │  └─ main.1a7488ce.css.map #z36y5v
-         └─ js
-            ├─ 788.bbd34b33.js #33pg04
-            ├─ 788.bbd34b33.js.LICENSE.txt #eplx8h
-            ├─ 788.bbd34b33.js.map #1xdy7n0
-            ├─ main.c2bf6fcb.js #3w772b
-            ├─ main.c2bf6fcb.js.map #1sdsrcm
-            ├─ runtime-main.b18f5039.js #qy2lhb
-            └─ runtime-main.b18f5039.js.map #1o0el5f"
+         ├─ js
+         │  ├─ 788.bbd34b33.js #33pg04
+         │  ├─ 788.bbd34b33.js.LICENSE.txt #eplx8h
+         │  ├─ 788.bbd34b33.js.map #1xdy7n0
+         │  ├─ main.f646b520.js #15ws1rq
+         │  ├─ main.f646b520.js.map #b2tlah
+         │  ├─ runtime-main.de012fdc.js #1qz643h
+         │  └─ runtime-main.de012fdc.js.map #ntuwq4
+         └─ media
+            └─ logo.103b5fa1..svg #1okqmlj"
```

Comment:
This logo.svg file being exported on its own is probably a result of this PR using the 'asset/resource' type.

If we instead use the 'asset' type, it would set an inline size limit of 8kb as I mentioned in my above comment and this logo file probably wouldn't be exported on its own.

Reply:
Resolved by moving to 'asset' type.